src/Simple.Migrations/DatabaseProvider/DatabaseProviderBase.cs

```diff
@@ -193,10 +193,10 @@ protected virtual void UpdateVersion(long oldVersion, long newVersion, string ne
                 versionParam.Value = newVersion;
                 command.Parameters.Add(versionParam);
 
-                var oldVersionParam = command.CreateParameter();
-                oldVersionParam.ParameterName = "OldVersion";
-                oldVersionParam.Value = oldVersion;
-                command.Parameters.Add(oldVersionParam);
+                //var oldVersionParam = command.CreateParameter();
```

Comment:
Is it right that these are now commented out?

Reply:
What is role of the parameter ? Is it used for any other provider ?

Reply:
It's there if custom database providers want to use it. Removing it would be a breaking change.